app/src/main/java/de/xikolo/App.kt

```diff
@@ -63,12 +73,17 @@ class App : Application() {
         instance = this
     }
 
+    @RequiresApi(Build.VERSION_CODES.N_MR1)
```

Comment:
Not sure if you wanna tell the compiler the whole `onCreate` method should be stripped before that API version.